### ISSUE TITLE
I am newbie in react and I have trouble with using redux !!! Please Help 

### ISSUE THREAD

**Issue by log901108** [mbrn/material-table#1238]
Hello, I am quite newb in react. I am struggling to understand by make toy projects. 
I faced trouble when I use material-table with redux. 
redux states in my application are well functioning before I entered a page which contains material-table.
>5 states : auth, user, loading, data, write

However, it changed totally when I enter the page which contains material-table.
>3 states : phase:"IDLE", completed:null, shouldFlush: false

Any guys know how to deal with it?

**Comment by log901108** [comment]
> my code is like below. I just used material-table in JSX. Without material-table redux goes fine.   
import React, { useEffect, useState, useCallback } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import {withRouter} from 'react-router-dom';
// @material-ui/core components
import { makeStyles, styled } from "@material-ui/core/styles";
import Hidden from "@material-ui/core/Hidden";
// core components
import GridItem from "components/Grid/GridItem.js";
import GridContainer from "components/Grid/GridContainer.js";
import Card from "components/Card/Card.js";
import CardHeader from "components/Card/CardHeader.js";
import CardBody from "components/Card/CardBody.js";
import Table from "components/Table/Table.js";

import {changeField, initialize} from '../../modules/write'
import styles from "assets/jss/material-dashboard-react/views/iconsStyle.js";

import MaterialTable from 'material-table';

const useStyles = makeStyles(styles);


function Form() {
 
	
  return (
    <GridContainer>  
       <GridItem xs={12} sm={12} md={12}>
        <Card plain>
        <CardHeader color="primary">
             
          </CardHeader>
          <CardBody>
			<MaterialTable/>
         
	
          </CardBody>
        </Card>
      </GridItem>
    </GridContainer>
  );
}

export default withRouter(Form);

**Comment by log901108** [comment]
>my code is below. redux goes fine when I delete material-table....

import React, { useEffect, useState, useCallback } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import {withRouter} from 'react-router-dom';
// @material-ui/core components
import { makeStyles, styled } from "@material-ui/core/styles";
import Hidden from "@material-ui/core/Hidden";
// core components
import GridItem from "components/Grid/GridItem.js";
import GridContainer from "components/Grid/GridContainer.js";
import Card from "components/Card/Card.js";
import CardHeader from "components/Card/CardHeader.js";
import CardBody from "components/Card/CardBody.js";
import Table from "components/Table/Table.js";

import {changeField, initialize} from '../../modules/write'
import styles from "assets/jss/material-dashboard-react/views/iconsStyle.js";

import MaterialTable from 'material-table';

const useStyles = makeStyles(styles);


function Form() {
 
	
  return (
    <GridContainer>  
       <GridItem xs={12} sm={12} md={12}>
        <Card plain>
        <CardHeader color="primary">
             
          </CardHeader>
          <CardBody>
			<MaterialTable/>
         
	
          </CardBody>
        </Card>
      </GridItem>
    </GridContainer>
  );
}

export default withRouter(Form);

**Comment by log901108** [comment]
>my code is below. redux goes fine when I delete material-table....

import React, { useEffect, useState, useCallback } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import {withRouter} from 'react-router-dom';
// @material-ui/core components
import { makeStyles, styled } from "@material-ui/core/styles";
import Hidden from "@material-ui/core/Hidden";
// core components
import GridItem from "components/Grid/GridItem.js";
import GridContainer from "components/Grid/GridContainer.js";
import Card from "components/Card/Card.js";
import CardHeader from "components/Card/CardHeader.js";
import CardBody from "components/Card/CardBody.js";
import Table from "components/Table/Table.js";

import {changeField, initialize} from '../../modules/write'
import styles from "assets/jss/material-dashboard-react/views/iconsStyle.js";

import MaterialTable from 'material-table';

const useStyles = makeStyles(styles);


function Form() {
 
	
  return (
    <GridContainer>  
       <GridItem xs={12} sm={12} md={12}>
        <Card plain>
        <CardHeader color="primary">
             
          </CardHeader>
          <CardBody>
			<MaterialTable/>
         
	
          </CardBody>
        </Card>
      </GridItem>
    </GridContainer>
  );
}

export default withRouter(Form);

**Comment by Domino987** [comment]
This has nothing to do with the table. It is probably something else. Can you create a [codesandbox](https://codesandbox.io/s/y2lrywpk21) to show us the problem?

**Comment by log901108** [comment]
Thank you for quick response. Unfortunately, I don't know how to make code sandbox. :(

However, there is some proceed in my project.
When I checked my states with redux-devtools(https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=ko) with chrome, it doesn't shows my original states. 
Meanwhile when I checked my states with react-devtools(https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=ko) shows my states appropriately. I don't know why it happened.
